### ISSUE TITLE
Use user defined functions compiled to WASM async module

### ISSUE THREAD

**Issue by Porsche9II** [handsontable/hyperformula#798]
### Description
It would be great to use functions from custom DLLs. Is this possible with hyperformula?

**Comment by wojciechczerniak** [comment]
What do you mean by that? Can you add more context?

**Comment by Porsche9II** [comment]
Is it possible to compile C++ code via emscripten to WASM and then use it via embind in hyperformula?
https://blog.esciencecenter.nl/using-c-in-a-web-app-with-webassembly-efd78c08469
Many companies already have a large codebase of UDFs written in C++ - it would be a nice options for those companies to get their existing code running with hyperformula...

**Comment by wojciechczerniak** [comment]
No, HF does not support this currently. WASM modules are run asynchronous so this would require implementing async UDFs first and is blocked by #9

A very good idea tho, thanks!
